common.mk

```diff
@@ -369,22 +373,17 @@ ifeq ($(CFG_TEE_BENCHMARK),y)
 filelist-tee-common: benchmark-app
 endif
 filelist-tee-common: fl:=$(GEN_ROOTFS_FILELIST)
-filelist-tee-common: optee-client xtest helloworld
-	@echo "# filelist-tee-common /start" 				> $(fl)
-	@echo "dir /lib/optee_armtz 755 0 0" 				>> $(fl)
+filelist-tee-common: optee-client xtest examples examples-filelist
+	@if [ ! -e $(fl) ]; then \
+		echo "# filelist-tee-common /start" > $(fl); \
+		echo "dir /lib/optee_armtz 755 0 0" >> $(fl); \
+	fi
```

Comment:
I think this is not very clean. optee_examples could store all generated CAs and TAs in a dedicated directories and leave to this `build/common.mk` to copy full content of these directories (if existing) to the target embedded filesystem directories /bin/ and /lib/optee_armtz/.